format/view-spec.md

```diff
@@ -160,6 +179,57 @@ Each entry in `version-log` is a struct with the following fields:
 | _required_  | `timestamp-ms` | Timestamp when the view's `current-version-id` was updated (ms from epoch) |
 | _required_  | `version-id`   | ID that `current-version-id` was set to |
 
+#### Storage Table Identifier
+
+The table identifier for the storage table that stores the precomputed results.
+
+| Requirement | Field name     | Description |
+|-------------|----------------|-------------|
+| _optional_  | `catalog`      | A string specifying the name of the catalog. If set to `null`, the catalog is the same as the view's catalog |
+| _required_  | `namespace`    | A list of strings for namespace levels |
+| _required_  | `name`         | A string specifying the name of the table/view |
+
+### Storage table metadata
```

Comment:
resurface a comment which might have been missed because it was combined with another minor comment. this is actually probably a little bigger decision. 

should this whole section be added to the `Table Spec` page? we can add cross references btw the `Table Spec` and `View Spec` pages

Reply:
I remember Dan saying that he would like to keep the Table spec clear of any MV structures. And I think it's good that all MV related information can be found in one place.

But I don't have a strong opinion on this and we could move it to the table spec.

Reply:
I am not too opinionated on this either. At least, I was wondering if this has been discussed and what does the community favor. 

I remember from previous discussion @wmoustafa  also favored all MV related info in one page. I can definitely understand why. We can keep it here.

Reply:
I wonder if it's still worth mentioning the fields that are only applicable to materialized view in appendix of table spec, e.g.  https://iceberg.apache.org/spec/#optional-snapshot-summary-fields and link to this page, so it is still searchable in the table spec page

Reply:
I support linking from Table => MV for this. I don't know if the information needs to be in both places

Reply:
I think a link is a great idea but I also agree that I would duplicate the information.

Reply:
@JanKaul can you add the link from table to this MV storage table section?